jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/ManifestPullerIntegrationTest.java

```diff
@@ -80,31 +112,41 @@ public void testPull_v22ManifestList() throws IOException, RegistryException {
         RegistryClient.factory(EventHandlers.NONE, "gcr.io", "distroless/base", httpClient)
             .newRegistryClient();
 
-    // Ensure ":latest" is a manifest list
+    // Ensures call to image at KNOWN_MANIFEST_LIST_SHA returns a manifest list
     V22ManifestListTemplate manifestListTargeted =
-        registryClient.pullManifest("latest", V22ManifestListTemplate.class).getManifest();
+        registryClient
+            .pullManifest(KNOWN_MANIFEST_LIST_SHA, V22ManifestListTemplate.class)
+            .getManifest();
     Assert.assertEquals(2, manifestListTargeted.getSchemaVersion());
     Assert.assertTrue(manifestListTargeted.getManifests().size() > 0);
 
-    // Generic call to ":latest" pulls a manifest list
-    ManifestTemplate manifestListGeneric = registryClient.pullManifest("latest").getManifest();
+    // Generic call to image at KNOWN_MANIFEST_LIST_SHA, should also return a manifest list
+    ManifestTemplate manifestListGeneric =
+        registryClient.pullManifest(KNOWN_MANIFEST_LIST_SHA).getManifest();
     Assert.assertEquals(2, manifestListGeneric.getSchemaVersion());
     MatcherAssert.assertThat(
         manifestListGeneric, CoreMatchers.instanceOf(V22ManifestListTemplate.class));
     Assert.assertTrue(((V22ManifestListTemplate) manifestListGeneric).getManifests().size() > 0);
+  }
 
-    // Referencing a manifest list by sha256, should return a manifest list
-    ManifestTemplate manifestListSha256 =
-        registryClient.pullManifest(KNOWN_MANIFEST_LIST_SHA).getManifest();
-    Assert.assertEquals(2, manifestListSha256.getSchemaVersion());
-    MatcherAssert.assertThat(
-        manifestListSha256, CoreMatchers.instanceOf(V22ManifestListTemplate.class));
-    Assert.assertTrue(((V22ManifestListTemplate) manifestListSha256).getManifests().size() > 0);
+  @Test
+  public void testPull_ociIndex() throws IOException, RegistryException {
+    RegistryClient registryClient =
+        RegistryClient.factory(EventHandlers.NONE, "gcr.io", "distroless/base", httpClient)
+            .newRegistryClient();
 
-    // Call to ":latest" targeting a manifest pulls a manifest.
-    V22ManifestTemplate manifestTargeted =
-        registryClient.pullManifest("latest", V22ManifestTemplate.class).getManifest();
-    Assert.assertEquals(2, manifestTargeted.getSchemaVersion());
+    // Ensures call to image at KNOWN_OCI_INDEX_SHA returns an OCI index
+    OciIndexTemplate manifestListTargeted =
+        registryClient.pullManifest(KNOWN_OCI_INDEX_SHA, OciIndexTemplate.class).getManifest();
```

Comment:
Alt: instead of `KNOWN_OCI_INDEX_SHA`, this test can keep using the "latest" tag (which keeps it susceptible for breakage but able to detect future changes in this image, as noted in https://github.com/GoogleContainerTools/jib/issues/3963#issuecomment-1481405637).